system_tests/pubsub.py

```diff
@@ -251,13 +262,15 @@ def test_fetch_delete_subscription_w_deleted_topic(self):
         subscription.create()
         topic.delete()
 
-        all_subs = []
-        token = None
-        while True:
-            subs, token = Config.CLIENT.list_subscriptions(page_token=token)
-            all_subs.extend(subs)
-            if token is None:
-                break
+        def _fetch():
+            return list(MethodIterator(Config.CLIENT.list_subscriptions))
+
+        def _found_orphan(result):
+            names = [subscription.name for subscription in result]
+            return ORPHANED in names
```

Comment:
Rather than staring at diff, try looking at the testcase as a whole:  `ORPHANED` is defined as obviously unique twenty lines above this point.